crates/compiler/fmt/src/header.rs

```diff
@@ -410,7 +411,14 @@ impl<'a> Formattable for ExposedName<'a> {
         indent: u16,
     ) {
         buf.indent(indent);
-        buf.push_str(self.as_str());
+        if buf.flags().snakify
+            && !self.as_str().is_empty()
+            && self.as_str().chars().nth(0).unwrap().is_ascii_lowercase()
```

Comment:
```suggestion
        if buf.flags().snakify
            && self.as_str().chars().first().is_some_and(|c| c.is_ascii_lowercase())
```

To avoid the failing "please don't add more panics" CI check

Reply:
Thanks, I thought about doing something like this, but that it might be harder to read. But I (not being a Rust pro) didn't know about is_some_and